src/service.ts

```diff
@@ -69,6 +69,7 @@ export interface ServiceOptions extends GoogleAuthOptions {
   promise?: PromiseConstructor;
   email?: string;
   token?: string;
+  timeout?: number; // http.request.options.timeout
```

Comment:
@alexander-fenster I'm curious - do we expose a gRPC option for timeout in the GAPIC libraries?

Reply:
Not directly, but GAPIC libraries can pass any gRPC option by prefixing it with `grpc.` and it will be passed all the way down to gRPC.